clinic/src/clj/clinic/fhir/utils.clj

```diff
@@ -0,0 +1,20 @@
+(ns clinic.fhir.utils)
+
+(defn find-code
+  "Returns the value of the `code` attribute corresponding to the given `system`
+   in a FHIR Codeable Concept."
+  [system codeable-concept]
+  (some-> codeable-concept
+          (get :coding)
+          ((partial filter #(= system (get % :system))))
+          (first)
+          (get :code)))
```

Comment:
Nothing wrong with this ^, but you _could_ also write it this way which is more readable imo
```
(some->> codeable-concept
     :coding
     (filter #(= system (:system %))
     first
     :code)
```